cylc/rose/entry_points.py

```diff
@@ -222,3 +228,44 @@ def record_cylc_install_options(
     )
     dumper(rose_suite_conf, rose_conf_filepath)
     return cli_config, rose_suite_conf
+
+
+def copy_config_file(
+    dir_=None,
+    opts=None,
+    dest_root=None,
+):
+    """Copy the ``rose-suite.conf`` from a workflow source to run directory.
+
+    Args:
+        dir_ (pathlib.Path | or str):
+            Source Path of Cylc install.
+        opts:
+            Not used in this function, but requried for consistent entry point.
+        dest_root (pathlib.Path | or str):
+            Destination path of Cylc install - the workflow rundir.
+
+    Return:
+        True if ``rose-suite.conf`` has been installed.
+        False if insufficiant information to install file given.
+    """
+    if (
+        dest_root is None or
+        dir_ is None
+    ):
+        raise FileNotFoundError(
```

Comment:
Before this returned False, which was causing failures in tests which didn't create both source and run directories. I think that this is safer.

Reply:
Tests passing locally for me now. Thanks Tim.